.github/workflows/linux.yml

```diff
@@ -1,6 +1,7 @@
 name: linux
 
 on:
+  workflow_dispatch:
```

Comment:
Why do we need this?

Reply:
I have added that, just to be able to manually enable the CI.